translib/app_utils_test.go

```diff
@@ -0,0 +1,124 @@
+////////////////////////////////////////////////////////////////////////////////
+//                                                                            //
+//  Copyright 2019 Broadcom. The term Broadcom refers to Broadcom Inc. and/or //
```

Comment:
It will be better to have all this test code in a dir by itself, so it does not clutter the main code.

Reply:
This is how go code is organized. It is not a clutter. Please check any standard go package --  https://github.com/golang/go/tree/master/src/strings.